test/Microsoft.ComponentDetection.Detectors.Tests/VcpkgComponentDetectorTests.cs

```diff
@@ -29,7 +29,7 @@ public void TestInitialize()
         [TestMethod]
         public async Task TestNlohmann()
         {
-            var spdxFile = @"{
+            var spdxFile = /*lang=json,strict*/ @"{
```

Comment:
What's this?

Reply:
Will look for more documentation about the `JSON002` error that raised the issue

Reply:
We spoke about this offline and decided to mark `JSON002` as a suggestion until we can find some documentation about it.